microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options.md

```diff
@@ -9,33 +9,40 @@ ms.subservice: extensions
 ms.date: 05/05/2023
 ---
 # Alternative ways to distribute an extension
+<!-- alts:
+# Use alternative installation methods 
+per https://developer.chrome.com/docs/extensions/how-to/distribute/install-extensions
+-->
 
 Generally, extensions are distributed through the Microsoft Edge Add-ons website. There are some scenarios where developers may need to distribute extensions using alternate methods. For example:
 
 1. The extension is associated with other software, and it should be installed together with the rest of the bundled software.
 
 1. Network administrators want to distribute an extension throughout their organization.
 
-Extensions that aren't loaded from the Edge Add-ons store are referred to as externally installed extensions. The following are alternate methods of distributing externally installed extensions:
+Extensions that aren't loaded from the Edge Add-ons store are referred to as _externally installed extensions_. The following are alternate methods of distributing externally installed extensions:
 
-*  Use the Windows registry (Windows only).
-*  Use a preferences JSON file (macOS and Linux).
+* Use the Windows registry (Windows only).
+* Use a preferences JSON file (macOS and Linux).
 
 These approaches are described below.
 
+See also:
+* [Set an externally installed extension to automatically update](../publish/auto-update.md)
+
 
 <!-- ====================================================================== -->
-## Before you begin
+## Gather initial information
 
-Make sure that you publish your extension in the Microsoft Edge Add-ons website, or package a `.crx` file and ensure that it installs successfully on your computer.  If you install the `.crx` file using the `update_url`, make sure you can go to your extension at that URL.
+Make sure that you publish your extension in the Microsoft Edge Add-ons website<!-- todo: but Intro says: "Generally, extensions are distributed through the Microsoft Edge Add-ons website. There are some scenarios where developers may need to distribute extensions using alternate methods." -->, or package a `.crx` file and ensure that it installs successfully on your computer.  If you install the `.crx` file using the `update_url`, make sure you can go to your extension at that URL.
```

Comment:
confusing whether this article is about this scenario or not: Using Partner Center to publish at Add-ons website.

rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/extensions-chromium/developer-guide/alternate-distribution-options?branch=pr-en-us-3269#gather-initial-information

Reply:
out of scope for present PR, keeping todo comment but closing PR Comment